tests/datacatalog_tag_manager/tag_datasource_processor_test.py

```diff
@@ -144,7 +173,7 @@ def test_create_tags_from_csv_permission_denied_lookup_entry_should_skip_resourc
         self.assertEqual('Test value', created_tag.fields['string_field'].string_value)
 
     def test_create_tags_from_csv_permission_denied_get_template_should_skip_template(
-        self, mock_read_csv):  # noqa
+            self, mock_read_csv):  # noqa
```

Comment:
Was this change performed by any tool? The current version is passing in the CI pipeline, so I didn't understand why we need to do this...

Reply:
I ran yapf, and it was formatted by it... weird, reverted it